notebooks/community/prediction/custom_prediction_routines/SDK_Pytorch_Custom_Predict.ipynb

```diff
@@ -0,0 +1,1852 @@
+{
```

Comment:
<p>use notebook template version for bucket</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/659/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Done.